client/src/pages/private/ParticipantTable.js

```diff
@@ -161,6 +164,14 @@ export default () => {
     }));
   };
 
+  const handleSiteSelector = (value) => {
+    setSiteSelector(value);
+    setPagination(oldPagination => ({
+      ...oldPagination,
```

Comment:
Not required to change anything but more of a conversation point, I've always seen this referred to as `prev` rather than giving it a new name, what do you think from a consistency POV?